dev-libs/libbpf/libbpf-9999.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs git-r3
+
+HOMEPAGE="https://www.kernel.org/doc/html/latest/bpf/bpf_devel_QA.html"
+DESCRIPTION="Stand-alone build of libbpf from the Linux kernel"
+EGIT_REPO_URI="https://github.com/libbpf/libbpf.git"
+
+
```

Comment:
2 blank lines in a row are forbidden in `::gentoo`

As a side note, changes do not need to be added as a separate PR, you can update the commit with `git commit --amend` then do a `git push --force` over the previous branch. Force pushing to your own fork is not rude or forbidden.